demos/theme/theme-yellow.scss

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "../../packages/mdc-theme/color-palette";
+
+$mdc-theme-primary: $material-color-yellow-500;
+$mdc-theme-secondary: $material-color-blue-500;
```

Comment:
Put a TODO here to call Sass mixins instead of using mdc-theme-primary and mdc-theme-secondary

Reply:
Done